src/runtime/basic-runtime-test.js

```diff
@@ -155,22 +163,6 @@ describes.realWin('installBasicRuntime', (env) => {
     await getBasicRuntime().whenReady();
     expect(progress).to.equal('123');
   });
-
-  it('should implement all APIs', async () => {
```

Comment:
Note: This test was essentially a copy of the "should implement BasicSubscriptions interface" test